README.rst

```diff
@@ -739,7 +739,7 @@ Note that if you use pytest `--strict` option, all bdd tags mentioned in the fea
 `markers` setting of the `pytest.ini` config. Also for tags please use names which are python-compartible variable
 names, eg starts with a non-number, underscore alphanumberic, etc. That way you can safely use tags for tests filtering.
 
-You can customize how hooks are converted to pytest marks by implementing the
+You can customize how tags are converted to pytest marks by implementing the
```

Comment:
Unrelated but probably not worth a separate PR - converting hooks to marks doesn't make any sense, so this should say "tags" instead.